.github/workflows/qc.yml

```diff
@@ -27,8 +27,6 @@ jobs:
       #----------------------------------------------
       - name: Install Poetry
         uses: snok/install-poetry@v1
-        with:
-          version: 1.3.2
```

Comment:
@hrshdhgd double check if right

Reply:
This deletion didn’t work

Reply:
It won't work. We have determined that `poetry==1.3.2` is the latest version that plays well with GH Actions. I'm not aware if the `poetry` developers have fixed their issues with GH actions but for now this works.

It seems like all the checks pass now.